client/reader/lib/header-section/index.tsx

```diff
@@ -1,16 +1,22 @@
-import { translate } from 'i18n-calypso';
+import { useTranslate } from 'i18n-calypso';
 import React from 'react';
 
-const renderHeaderSection = () => (
-	<div className="logged-out-reader-header">
-		<h2>
-			{
-				// translators: The title of the reader tag page
-				translate( 'WordPress Reader' )
-			}
-		</h2>
-		<h1>{ translate( 'Enjoy millions of blogs at your fingertips.' ) }</h1>
-	</div>
-);
+const ReaderHeaderSection = () => {
+	const translate = useTranslate();
+
+	return (
+		<div className="logged-out-reader-header">
+			<h2>
+				{
+					// translators: The title of the reader tag page
+					translate( 'WordPress Reader' )
+				}
+			</h2>
+			<h1>{ translate( 'Enjoy millions of blogs at your fingertips.' ) }</h1>
+		</div>
+	);
+};
+
+const renderHeaderSection = () => <ReaderHeaderSection />;
 
 export default renderHeaderSection;
```

Comment:
Is this indirection necessary? Can we just export `ReaderHeaderSection`?

```suggestion
export default ReaderHeaderSection;
```

Reply:
Good point. I don't think the indirection is strictly necessary, but I noticed that the exported value is exclusively used as `context.renderHeaderSection = renderHeaderSection;` across the Reader. I thought that by exporting `ReaderHeaderSection` directly, we might lose the context that it's meant to be assigned to `context.renderHeaderSection`. But we could add that as a comment, of course.

Just in case the `ReaderHeaderSection` name resembled `renderHeaderSection` too much, I renamed the component to `ReaderHeader`.

If my reasoning is unnecessary and we'd best just export `ReaderHeaderSection`, let me know!

Reply:
Feel free to leave it as-is, but without further research, I expect this has something to do with ancient CJS code remnants and doesn't have a particular purpose. It is not a blocker for the PR by any means.

Reply:
Thanks, that makes sense! I'll remove the indirection in a separate PR, as it would probably be best to rename the related imports too, which would require changes to a couple of Reader sub-packages and add a bit of noise to this PR.